WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsWCPayNotSetup.swift

```diff
@@ -1,6 +1,8 @@
 import SwiftUI
+import Yosemite
 
 struct InPersonPaymentsWCPayNotSetup: View {
```

Comment:
It would be worth renaming this I think

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5958/commits/a896d98d7f8a1245ba680fd6134301424ef0febe